### PR TITLE
Feature/acceptance test fixes

### PR DESCRIPTION
**Pull request by jcalderin**
Modifications an fixes to get the project executed in jenkins

### CODE REVIEW

File: test/acceptance/integration/terrain.py [hunk 1]
```diff
@@ -60,7 +60,7 @@ def before_all_scenarios():
     user_roles_general = [(world.ks['user_all'], x['name']) for x in
                           world.ks['environment_general']['domains'][0]['users'][0]['projects'][0]['roles']]
     initialize_ac(user_roles_general,
-                  world.ac['ip'],
+                  world.ac['ip'], world.ac['port'],
```
Comment (dmoranj) [inline test/acceptance/integration/terrain.py]: Do you think having two attributes in the same line is compliant with Python's style guidelines? (I'm not an expert in the matter, but I'd introduce a break there).

File: test/acceptance/tools/deploy_pep.py [hunk 2]
```diff
@@ -227,3 +227,66 @@ def stop_pep(ip, user, password, port='22'):
         for proc_pid in pid.split('\n'):
             sudo('kill -9 {pid}'.format(pid=proc_pid.strip()))
 
+
+def start_pep_local(pep_path='/fiware-orion-pep'):
+    """
+    Given a ssh connection data, stop PEP if its running, put the new configuration, and start is.
+    The machina have to has the "dtach" package
```
Comment (dmoranj) [inline test/acceptance/tools/deploy_pep.py]: "Has to have" instead of "have to has"

**Comment by dmoranj** [comment]
Apart from minor comments, LGTM

File: test/acceptance/tools/deploy_pep.py (hunk 2 above)
Comment (jcalderin) [inline test/acceptance/tools/deploy_pep.py]: Fixed in 70180e0

File: test/acceptance/integration/terrain.py (hunk 1 above)
Comment (jcalderin) [inline test/acceptance/integration/terrain.py]: NTC - Python style guidelines tells that a line has to be less than 80 characters, but nothing about this. Its done deliberately to group the kind of parameters, to be more readable

**Pull request merged**